### DISCUSSION TITLE
Merge Multiple NFT

### DISCUSSION THREAD

**Discussion by moechenche** [Chia-Network/chia-blockchain#7166] [Q&A]
When creating pool plots I unintentionally created 6 NFTs, each having 1 plot.  Is there a way to merge these all into one or just delete the 5 i don't want?  I deleted them from the config.yaml on mainnet but after 10 minutes or so they reloaded on the GUI and on the config.yaml.  They must be stored somewhere else.

**Comment by CryptoBlockchainTechnologies** [comment]
As of now and probably forever, there is no way to get rid of an NFT tied to a wallet. The request to the dev team has been to be able to filter your NFTs displayed on the GUI. So basically you would set all the ones you don't want to see as solo plots and then set a filter to hide solo plots. There are other ways to hide them without having to change them to solo; hide inactive NFTs, hide 0 plot NFTs, etc... These are all possible cosmetic features that can be added later. All depends on priority by dev team.

**Reply by mariano54** [reply]
Yeah this is correct.

**Reply by eoun** [reply]
I assume that ii is also not possible (at least not at the moment) to reassign a (portable) plot to a new NFT.

The Story: I run to computers (with the same key and during testing on the 1.1.797 Version). I have joined a pool on one computer but the pool does not show up on the other one, I was not thinking anything was wrong. And so I plotted some, and these worked fine. Now I wanted to join a pool on the second computer. This worked fine as well but now this pool also shows up in the other computer (they both have the same Launcher Id). So now I have two NFTs in one of the computers and I want to move all the plots that a have done to the pool that shows up at both computers. But I cant find a way how I can delete a NFT or assign plots to a pool. (As I know now is currently not possible). I want to make this so that al my plots are managed from one Launcher Id, so that every think is better to handle for me. And also should both pool Overview show the same or not (regarding the number of plots and points (so shows the Overview only the plots on his machine or the other one too))

**Reply by loppefaaret** [reply]
you cant move plots from one plot NFT to another - they are belonging to a specific plot NFT
